simulator/simulator.py

```diff
@@ -0,0 +1,66 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+import datetime
+import time
+import json
+import secrets
+from pprint import pprint
```

Comment:
```suggestion
import datetime
import json
import secrets
import time
from pprint import pprint

from firebase_admin import credentials, db, initialize_app
```

Suggest follow PEP8 guide to re-order the imports.

Reply:
addressed in [012318c](https://github.com/ricky9667/smart_space/pull/14/commits/012318ca135c29268f587681ab608a6983f027da).